.github/workflows/full-rebuild.yml

```diff
@@ -0,0 +1,169 @@
+name: Re-build and publish All
+on:
+  push:
+    paths:
+        - .github/workflows/full-rebuild.yml
+        - 'AlmaLinux9/Dockerfile-spack'
+  workflow_dispatch:
+
+env:
+  REGISTRY: ghcr.io
+
+permissions:
+  contents: read
+  packages: write
+  id-token: write
+  attestations: write
+
+jobs:
+  build-spack:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [{dir: AlmaLinux9,suffix: alma9}]
+    env:
+      IMAGE_NAME: mucoll-spack
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Set up Docker buildx
+        uses: docker/setup-buildx-action@v3
+      - name: Login to container registry
+        uses: docker/login-action@v3
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Make owner name lowercase
+        run: |
+            echo "OWNER_LC=${OWNER,,}" >>${GITHUB_ENV}
```

Comment:
Did you find that it was required to lowercase `github.repository` for this to work?

Reply:
Yes, the GitHub registry doesn't like capital lerters